File: prototype/fix.js
// Quick fix script: read FACING_ACTIONS section
const fs = require('fs');
const content = fs.readFileSync('C:\\Users\\michaellhao\\CodeBuddy\\DPoker\\prototype\\index.html', 'utf8');
const idx = content.indexOf('FACING_ACTIONS');
if (idx >= 0) {
  console.log("FOUND at index " + idx);
  console.log(JSON.stringify(content.substring(idx, 400)));
} else {
  console.log("NOT FOUND");
}

File: prototype/fix.py
import os, json

filepath = r'C:\Users\michaellhao\CodeBuddy\DPoker\prototype\index.html'
with open(filepath, 'r', encoding='utf-8') as f:
    content = f.read()

idx = content.find('FACING_ACTIONS')
if idx >= 0:
    print(f"FOUND at index {idx}")
    print(repr(content[idx:idx+500]))
else:
    print("NOT FOUND")
